pkg/cmd/target/unset.go

```diff
@@ -21,7 +21,12 @@ import (
 func NewCmdUnset(f util.Factory, o *UnsetOptions) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "unset",
-		Short: "Unset target, e.g. \"gardenctl target unset shoot\" to unset currently targeted shoot",
+		Short: "Unset target",
+		Example: `# unset selected shoot
+gardenctl target unset shoot
+
+# unset garden and everything targeted inside it
```

Comment:
Maybe it's better to be more explicit. Like "Unset garden. This will also unset a targeted project, shoot, seed and control plane" or something like that.

Reply:
ok. makes sense I guess.